src/ec/packages/ec-component-fact-figures/ecl-fact-figures.html.twig

```diff
@@ -80,7 +80,7 @@
   </div>
 {% endif %}
 
-{% if view_all.link is defined and view_all.link is not empty %}
+{% if view_all.link is defined and view_all.link is not empty and view_all.visible == 1  %}
```

Comment:
like before

Reply:
Also in this case i would first create a local variable:
[{% set _view_all_visibility = view_all.visible|default(false) %}](url)
and then you would check for `and _view_all.visibility`

Reply:
There is a double space at the end of line 83, this means that you still don't have grumphp running when you commit, @Joosthe , can you please verify this? Otherwise we would have this problem everytime.

Reply:
On line 34 there are two spaces at the end of the line, also

Reply:
I don’t really understand the usefulness of this _view_all.visible_ parameter. if the link should not appear, I think that we just have to remove the view_all object?

Reply:
Indeed i agree that is redundant, it's not entirely clear the reason why one would define this view_all object if the intention is not to show the link it generates.
I thought the same parameter existed in ECL and that's why we were adding it on our side, but in ECL it doesn't exist, so i agree about simply removing it.

Reply:
ok I'll update, thx